conf/app.routes

```diff
@@ -5,11 +5,11 @@
 # Home page
 GET    /                                 controllers.SelfServiceTimeToPayController.present()
 POST   /                                 controllers.SelfServiceTimeToPayController.submit()
-#GET    /eligibility                     controllers.EligibilityController.present()
-#GET    /eligibility/debt-type           controllers.EligibilityController.debtTypePresent()
-#POST   /eligibility/debt-type           controllers.EligibilityController.debtTypeSubmit()
-#GET    /eligibility/existing-ttp        controllers.EligibilityController.existingTtpPresent()
-#POST   /eligibility/existing-ttp        controllers.EligibilityController.existingTtpSubmit()
+GET    /eligibility                     controllers.EligibilityController.present()
+GET    /eligibility/debt-type           controllers.EligibilityController.debtTypePresent()
+POST   /eligibility/debt-type           controllers.EligibilityController.debtTypeSubmit()
+GET    /eligibility/existing-ttp        controllers.EligibilityController.existingTtpPresent()
+POST   /eligibility/existing-ttp        controllers.EligibilityController.existingTtpSubmit()
```

Comment:
All the eligibility routes, models, controllers and pages should be removed as they are no longer part of the non-logged in journey flow.